awesome_dashboard/static/src/dashboard/config_dialog.js

```diff
@@ -0,0 +1,44 @@
+import { Component, useState } from "@odoo/owl";
+import { Dialog } from "@web/core/dialog/dialog";
+import { items } from "./dashboard_items";
+import { registry } from "@web/core/registry";
+import { CheckBox } from "@web/core/checkbox/checkbox";
+import { browser } from "@web/core/browser/browser";
+
+
+export class ConfigDialog extends Component {
+    static template = "awesome_dashboard.ConfigDialog";
+    static components = { Dialog, CheckBox }; 
+    static props = ["close", "items", "disabledItems", "onUpdateConfigs"];
+    setup() {
+        // create object for each item with an enabled property
+        this.items = useState(this.props.items.map((item) => {
+            return {
+                ...item,
+                enabled: !this.props.disabledItems.includes(item.id),
+            }
+        }));
+        
+    }
+    
+    // adds the unchecked item to the disabled list
```

Comment:
The list is not disabled though
```suggestion
    // Add unchecked item to disabled items list
```
It's also important to be concise with your comments. After you write a sentence always try to remove 'the' from the sentence and keep it like that if the sentence still makes sense 🙂